Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs

```diff
@@ -0,0 +1,134 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Remotion.Utilities;
+
+namespace Remotion.Web.Development.WebTesting.Accessibility.Implementation
+{
+  /// <inheritdoc cref="IAccessibilityResultMapper" />
+  public class AccessibilityResultMapper : IAccessibilityResultMapper
+  {
+    /// <inheritdoc />
+    public AccessibilityResult Map (AxeResult deserializedResult)
+    {
+      ArgumentUtility.CheckNotNull ("deserializedResult", deserializedResult);
+
+      return new AccessibilityResult (
+          axeVersion: deserializedResult.TestEngine.Version,
+          url: deserializedResult.Url,
+          timestamp: ParseTimestamp (deserializedResult.Timestamp),
+          windowWidth: deserializedResult.TestEnvironment.WindowWidth,
+          windowHeight: deserializedResult.TestEnvironment.WindowHeight,
+          orientationAngle: deserializedResult.TestEnvironment.OrientationAngle,
+          orientationType: deserializedResult.TestEnvironment.OrientationType,
+          userAgent: deserializedResult.TestEnvironment.UserAgent,
+          includeIFrames: deserializedResult.ToolOptions.IFrames,
+          conformanceLevel: ParseConformanceLevel (deserializedResult.ToolOptions.RunOnly.Values),
+          violations: ParseViolations (deserializedResult).ToList());
+    }
+
+    private DateTime ParseTimestamp (string timestampAsString)
+    {
+      return DateTime.ParseExact (
+          timestampAsString,
+          "yyyy-MM-ddTHH:mm:ss.fffZ",
+          CultureInfo.CreateSpecificCulture ("en-US"),
+          DateTimeStyles.AdjustToUniversal);
+    }
+
+    private AccessibilityConformanceLevel ParseConformanceLevel (string[] tags)
+    {
+      var conformanceLevels = tags.Where (AccessibilityConformanceLevelConverter.IsValid);
+
+      if (tags.Contains ("section808"))
```

Comment:
It must have been a typo, either way, this if statement has no effect.

What should be the behavior when both (any) WCAG & Section508 are requested conformance levels? Should this be allowed?
What should be the (singular) conformance level in the output? Or should it be changed to an `IReadOnlyList`?

Reply:
I think we decided to not support enabling section508. anyway, right now, we only support doing one conformance level at a time since that's what we actually need.

Reply:
> I think we decided to not support enabling section508

I added dc10c549effed25febf7b77ea58d9d9b77feb6a5.

Reply:
yes, separte enums could make sense. depends a bit on just how well we can name things, how much overhead this incurs in terms of code, etc.